web/geo/generateZoneBoundingBoxes.ts

```diff
@@ -6,117 +6,139 @@ import { saveZoneYaml } from './files.js';
 import { getJSON } from './utilities.js';
 import { WorldFeatureCollection, ZoneConfig } from './types.js';
 import { Position } from '@turf/turf';
-
-const inputArguments = process.argv.slice(2);
+import { mergeZones } from '../scripts/generateZonesConfig.js';
 
 const zonesGeo: WorldFeatureCollection = getJSON(
   path.resolve(fileURLToPath(new URL('world.geojson', import.meta.url)))
 );
 
-if (inputArguments.length <= 0) {
-  console.error(
-    'ERROR: Please add a zoneName parameter ("ts-node --esm generateZoneBoundingBoxes.ts DE")'
+function generateBoundingBox(zoneKey: string) {
+  const zonePath = path.resolve(
+    fileURLToPath(new URL(`../../config/zones/${zoneKey}.yaml`, import.meta.url))
   );
-  process.exit(1);
-}
+  const zoneConfig = yaml.load(fs.readFileSync(zonePath, 'utf8')) as ZoneConfig;
 
-const zoneKey = inputArguments[0];
-
-const zonePath = path.resolve(
-  fileURLToPath(new URL(`../../config/zones/${zoneKey}.yaml`, import.meta.url))
-);
-const zoneConfig = yaml.load(fs.readFileSync(zonePath, 'utf8')) as ZoneConfig;
+  if (!zoneConfig) {
+    console.error(`ERROR: Zone ${zoneKey} does not exist in configuration`);
+    process.exit(1);
+  }
 
-if (!zoneConfig) {
-  console.error(`ERROR: Zone ${zoneKey} does not exist in configuration`);
-  process.exit(1);
-}
+  // do not modifiy current bounding boxes
+  if (zoneConfig.bounding_box) {
+    console.error(`Ignoring ${zoneKey} as it already have a bounding box`);
+    return;
+  }
```

Comment:
Why do we want to ignore existing zones? If they have changed the bouding box should also be updated (like with the texas change).